orc8r/cloud/go/obsidian/swagger/protos/swagger.proto

```diff
@@ -17,8 +17,11 @@ option go_package = "protos";
 
 // SwaggerSpec provides a Swagger specification of the service's REST API.
 service SwaggerSpec {
- // GetSpec returns the Swagger specification of the service's REST API.
- rpc GetSpec(GetSpecRequest) returns (GetSpecResponse) {}
+ // GetPartialSpec returns partial Swagger specification of the service's REST API.
+ rpc GetPartialSpec(GetSpecRequest) returns (GetSpecResponse) {}
```

Comment:
Each Get*Spec should have its own unique req+res message definitions

Reply:
Besides the benefit of just having separated definitions which may be useful for flexible development in the future, is there a good reason why using shared definitions is bad? Especially considering how simple and identical in functionality they are?